src/NBitcoin/ChainedHeader.cs

```diff
@@ -4,6 +4,60 @@
 
 namespace NBitcoin
 {
+    /// <summary>
+    /// Represents the state of the availability of a block.
+    /// </summary>
+    public enum BlockDataAvailabilityState
+    {
+        /// <summary>
+        /// A <see cref="BlockHeader"/> is found and is a valid header.
```

Comment:
I would not say "found". I would also not say that it is a valid header, because these states are about availability of the data. This means that this particular state represents having data of the header.

Reply:
I would also say that this is an initial value and that is because before we have full data of the header, we are unable to construct chained header.